### PR TITLE
Deliver OpenSquare paid QA spec 1.0 implementation

### PR DESCRIPTION
**Pull request by wliyongfeng**
# Milestone Delivery Checklist

- [x] The [milestone-delivery-template.md](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/deliveries/milestone-delivery-template.md) has been copied and updated.
- [x] The [invoice form :pencil:](https://forms.gle/LSRr7PCjBpEbKGh89) has been filled out for this milestone.
- [x] This pull request is being made by the same account as the accepted application.
- [x] In case of acceptance, the payment will be transferred to the BTC/ETH payment address provided in the application.
- [x] The delivery is according to the [Guidelines for Milestone Deliverables](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).

Link to the application pull request: https://github.com/w3f/Grants-Program/pull/786

### CODE REVIEW

**Comment by lucasvanmol** [comment]
Hey @wliyongfeng, could you provide instructions on how to setup a local ipfs node instead of using infura? Infura requires entering credit card information even for free projects, so I'd rather setup a local ipfs node instead.

**Review by lucasvanmol — CHANGES_REQUESTED** [review]
You seem to have some typos here, could you please fix them? Thanks

File: deliveries/OpenSquare_paid_qa-milestone_1.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+| 0a. | License | https://github.com/opensquare-network/paid-qa/blob/main/LICENSE | Apache 2.0 |
+| 0b. | Code-1 | https://github.com/opensquare-network/paid-qa/tree/release-1.0.0 | |
+| 0c. | Code-2 | https://github.com/opensquare-network/paid-qa/tree/release-1.0.0/packages/scan | |
+| 0d. | Code-3 | [Repo for IPFS client](https://github.com/opensquare-network/ipfs); [Script to pin file to IPFS](https://github.com/opensquare-network/paid-qa/blob/release-1.0.0/packages/server/src/scripts/pin-to-ipfs.js); [Scripts to fetch file from IPFS](https://github.com/opensquare-network/paid-qa/tree/release-1.0.0/packages/scan/src/ipfs) | |
+| 0e. | Code-4 | [Script to pin topics/answers](https://github.com/opensquare-network/paid-qa/blob/release-1.0.0/packages/server/src/scripts/pin-to-ipfs.js); [Code which submits batch of answers](https://github.com/opensquare-network/paid-qa/blob/release-1.0.0/packages/node-api/src/features/remark/remark.controller.js#L163) | |
+| 0c. | Documentation | https://github.com/opensquare-network/paid-qa/discussions/697 | [spec repo](https://github.com/opensquare-network/qa-spec) and another [tutorial](https://hackmd.io/@WIq3uV0rRwS6W3GGj2pmyA/SJKC9ucjc) |
```
[the remark this answers is not in the document]
Comment (wliyongfeng) [reply deliveries/OpenSquare_paid_qa-milestone_1.md]: Fixed, thank you.

**Comment by lucasvanmol** [comment]
Hey there, I'm trying to visit https://review-paid-qa.on.fleek.co/ again, but am being met with the following, even though it was working fine yesterday:
Screenshot 2022-07-14 at 11 27 57

**Comment by wliyongfeng** [comment]
> Hey there, I'm trying to visit https://review-paid-qa.on.fleek.co/ again, but am being met with the following, even though it was working fine yesterday

Weird, it works well on my side. Ok, I'll deploy another environment. I'll let you know when it's ready.

**Comment by wliyongfeng** [comment]
@lucasvanmol Please visit https://review-paid-qa.opensquare.io if https://review-paid-qa.on.fleek.co is still unavailable.

**Comment by lucasvanmol** [comment]
@wliyongfeng Thanks a lot for this! It looks like I can connect to both just fine now, however, so it may have just been a temporary problem. I'm now trying to run the project locally, but running into some problems: I'll share them in the matrix opensquare matrix channel.

**Comment by wliyongfeng** [comment]
> @wliyongfeng Thanks a lot for this! It looks like I can connect to both just fine now, however, so it may have just been a temporary problem. I'm now trying to run the project locally, but running into some problems: I'll share them in the opensquare matrix channel.

Have replied in the cannel. It's a MongoDB [replica set](https://www.mongodb.com/docs/manual/tutorial/convert-standalone-to-replica-set/) setting problem. We need this to keep some business transactional and will add this configuration warning to the README.

**Comment by lucasvanmol** [comment]
Thanks again @wliyongfeng for the assisstance on element. I've accepted the delivery - please take a look at my final evaluation [here](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/evaluations/opensquare-paid-qa_m1_lucasvanmol.md). I'll also forward your invoice internally.

**Comment by RouvenP** [comment]
hi @wliyongfeng, we transferred the payment yesterday. Thanks!

**Comment by wliyongfeng** [comment]
> hi @wliyongfeng, we transferred the payment yesterday. Thanks!

@RouvenP Confirmed. Thanks!

**Pull request merged**